src/data/map.js

```diff
@@ -607,7 +607,7 @@ const getPokestops = async (minLat, maxLat, minLon, maxLon, updated = 0, showPok
     SELECT id, lat, lon, name, url, enabled, lure_expire_timestamp, last_modified_timestamp, updated,
             quest_type, quest_timestamp, quest_target, CAST(quest_conditions AS CHAR) AS quest_conditions,
             CAST(quest_rewards AS CHAR) AS quest_rewards, quest_template, cell_id, lure_id, pokestop_display,
-            incident_expire_timestamp, grunt_type, sponsor_id
+            incident_expire_timestamp, grunt_type, sponsor_id, ar_scan_eligible
```

Comment:
```suggestion
            incident_expire_timestamp, grunt_type, sponsor_id,${ar_scan_eligible}
```

Reply:
Probably want to remove the `,` since we're including it in the `ar_scan_eligible` const
```
incident_expire_timestamp, grunt_type, sponsor_id ${ar_scan_eligible}
```
Also suggesting `arScanEligible` instead of `ar_scan_eligible` for variable name